src/GraphQL.Tests/Serialization/NewtonsoftJson/InputsConverterTests.cs

```diff
@@ -0,0 +1,302 @@
+using System.Collections.Generic;
+using GraphQL.NewtonsoftJson;
+using Newtonsoft.Json;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests.Serialization.NewtonsoftJson
+{
+    public class InputsConverterTests
```

Comment:
These tests are pretty much an exact clone of the tests that already existed within `ObjectDictionaryConverterFacts` for `SystemTextJson`.  I made only the necessary changes so it would work with the Newtonsoft serializer.